Add instruction on windows matching

soraxas opened the pull request:
I was trying to find a way to put a window that has no defined `wm_class` to scratch layer. I had thought of using `title` to match the window and only realised that it had already been implemented without being mentioned in the README.

This PR adds some hints to such an approach for window matching.

soraxas committed:
Add instruction on windows matching

I was trying to find a way to put a window that has no defined `wm_class` to scratch layer. I had thought of using `title` to match the window and only realised that it had already been implemented without being mentioned in the README.

This PR adds some hints to such an approach for window matching.

smichel17 reviewed (approved):
Looks good, but let's merge this after #415, so it's working better before we put more eyes on it.

smichel17 committed:
Fix crash in scratch layer matching

If the user tried to match only on wm_title instead of both title &
class, there would be an error when trying to access
prop.wm_class.constructor (since prop.wm_class is undefined).

To fix this, check for a RegExp using instanceof instead (for wm_title,
too).

smichel17 commented:
Thanks, and sorry for taking so long to get around to this (but hey, at shorter than it's been sometimes :)
